### PR TITLE
Fifth (and final for this quarter) round of documented nodes for review

### PR DESCRIPTION
**Pull request by johnpierson**
The first round of selections category for node help in the documentation browser.

Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose
To add node help for Revit nodes

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] Snapshot of UI changes, if any.

### Reviewers

@hwahlstrom , @lillismith 

### FYIs

@QilongTang

### CODE REVIEW

File: doc/distrib/NodeHelpFiles/Revit.Elements.FamilyParameter.GroupType.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## In Depth
+`FamilyParameter.GroupType` reports the group type of a given parameter. A parameters group type controls where it appears in the property palette in Revit.
```
Comment (hwahlstrom) [inline doc/distrib/NodeHelpFiles/Revit.Elements.FamilyParameter.GroupType.md:2]: Add hyphen: "A parameter's group type"

File: doc/distrib/NodeHelpFiles/Revit.Elements.FamilyParameter.GroupType.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+In the example below, the current Revit document is converted to a family document and all the parameters are obtained. `FamilyParameter.GroupType` is used to obtain the groupt type of all of the parameters in the family.
```
Comment (hwahlstrom) [inline doc/distrib/NodeHelpFiles/Revit.Elements.FamilyParameter.GroupType.md:4]: Extra t after "group" in "used to obtain the group type"

File: doc/distrib/NodeHelpFiles/Revit.Elements.FamilyParameter.Name.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+## In Depth
+`FamilyParameter.Name` obtains the user visible name from the given famly parameter element.
```
Comment (hwahlstrom) [inline doc/distrib/NodeHelpFiles/Revit.Elements.FamilyParameter.Name.md:2]: Typo: "famly"

File: doc/distrib/NodeHelpFiles/Revit.Schedules.ScheduleFilter.FilterType.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+## In Depth
+`ScheduleFilter.FilterType` returns the method used for the input filter.
+Possible filter types include:
+
+- Equal - The field value is equal to the specified value.
+- NotEqual - The field value is not equal to the specified value.
+- GreaterThan - The field value is greater than the specified value.
+- GreaterThanOrEqual - The field value is greater than or equal to the specified value.
+- LessThan - Less The field value is less than to the specified value.
```
Comment (hwahlstrom) [inline doc/distrib/NodeHelpFiles/Revit.Schedules.ScheduleFilter.FilterType.md:9]: Extra "Less" before "The field value"?
Also remove "to" from "less than to the specified value"

File: doc/distrib/NodeHelpFiles/Revit.Schedules.ScheduleFilter.FilterType.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+- Contains - For a string field, the field value contains the specified string.
+- NotContains - For a string field, the field value does not contain the specified string.
+- BeginsWith - For a string field, the field value begins with the specified string.
+- NotBeginsWith - For a string field, the field value does not begin with the specified string.
+- EndsWith - For a string field, the field value ends with specified string.
+- NotEndsWith - For a string field, the field value does not end with the specified string.
+- IsAssociatedWithGlobalParameter - The field is associated with a specified global parameter of a compatible type
+- IsNotAssociatedWithGlobalParameter - The field is not associated with a specified global parameter of a compatible type 
+
+In the example below, the first schedule from the current Revit file is collected. The schedule view is then checked for filters, and the only filter applied is a string does not end with filter type.
```
Comment (hwahlstrom) [inline doc/distrib/NodeHelpFiles/Revit.Schedules.ScheduleFilter.FilterType.md:20]: "..is a string **that** does not end with filter type"?

**Review by hwahlstrom — APPROVED** [review]
LGTM with small fixes

File: doc/distrib/NodeHelpFiles/Revit.Schedules.ScheduleFilter.FilterType.md (hunk 4 above)
Comment (johnpierson) [reply doc/distrib/NodeHelpFiles/Revit.Schedules.ScheduleFilter.FilterType.md:9]: Oops this was copied directly from the API docs, so I might let them know too. lol

File: doc/distrib/NodeHelpFiles/Revit.Schedules.ScheduleFilter.FilterType.md (hunk 5 above)
Comment (johnpierson) [reply doc/distrib/NodeHelpFiles/Revit.Schedules.ScheduleFilter.FilterType.md:20]: revised to "string does not end with" as that is the rule name

**Pull request merged**